perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -222,6 +223,7 @@ sub report_form_ajax : Path('ajax') : Args(0) {
 
     $c->stash->{json_response} = {
         bodies          => \@list_of_names,
+        display_names   => \%display_names,
```

Comment:
This adds weight to every single ajax call, for currently one very specific usage. At the least could it be non-existent if empty, and only contain the row for which it needs to be there?

Reply:
d4b788f28075bb6566808c402b148ea58fac16f8 ensures only names that are different are included in the `display_names` output, and doesn't include that key in the JSON if it would be empty.